perllib/FixMyStreet/Cobrand/Hounslow.pm

```diff
@@ -73,23 +73,18 @@ sub link_to_council_cobrand { "Hounslow Highways" }
 # Instead, force the borough council name to be used.
 sub all_reports_single_body { { name => "Hounslow Borough Council" } }
 
-sub open311_pre_send {
-    my ($self, $row, $open311, $h) = @_;
+sub open311_post_send {
+    my ($self, $row, $h) = @_;
 
     return unless $row->isa("FixMyStreet::DB::Result::Problem");
```

Comment:
No need for this as post_send is only called on problems, unlike pre_send, but won't affect anything.